build.assets/Dockerfile-centos7

```diff
@@ -1,9 +1,9 @@
-ARG RUST_VERSION
+# syntax=docker/dockerfile:1
 
-## GIT2 ###################################################################
+ARG RUST_VERSION
 
-# git2 packages are not available on ARM64, so we need to build it from source.
-FROM centos:7 AS git2
+# Set up a base container from centos:7 adjusted for using valut.centos.org
```

Comment:
nit: small typo
```suggestion
# Set up a base container from centos:7 adjusted for using vault.centos.org
```

Reply:
I typo'ed that word pretty much every time I typed it. Thanks for catching it. Fixed here and in the fips Dockerfile.